api/peripheral_api/util/LedSign.js

```diff
@@ -0,0 +1,34 @@
+const axios = require('axios');
+
+async function updateSign(req) {
+  return new Promise((resolve) => {
+    try {
```

Comment:
the try catch is unnecessary here because youre already doing `.catch()` on the axios call. remove for both